vchuravy/ScopedValues.jl
# Try to allow abstract scopes and add dict scope

- Set version to 1.0.0
- Print scope
- tests
- add back deep tests
- check error on conversion
- doc string on current scope
- Update Project.toml
- use recursion -- slow
- add IdDict scope
